### ISSUE TITLE
Home bridge with OS Yosemite and Raspian Jessie (Raspberry Pi 2 B)

### ISSUE THREAD

**Issue by raspman** [cflurin/homebridge-punt#5]
Hi,

I've been working on this for days and simply can't figure out what the issue is.  I can (sometimes) get home bridge to work when running off my OS Yosemite.  I can see the HAP Tcp using the methods described, however, sometimes Eve (the iOS App I'm using) won't recognize any devices.  Once I start playing around with the name (e.g. Home-02,) I can sometimes get Eve to recognize it again.  HOWEVER, running everything on Raspian Jessie, I cannot see the HAP Tcp on the network.  I can get home bridge to run fine, all the way up to the numbers it gives you that you need to plug into the iOS app. (Eve,) but I simply can't get the HAP Tcp to show up and have Eve recognize everything.  I'm guessing it is some type of network setting issue?  I can ssh to my raspberry with no problem. 

Thank you for your help.

**Comment by cflurin** [comment]
Hi raspman,

Could you describe your LAN/WLAN Layout.
How is your OS X device connected to the router? ethernet or WiFi?
Are you using an additional Access Point device?
Have you a WIFI-adapter for your RPi or are you using the build-in ethernet?

**Comment by raspman** [comment]
Hi,

Thank you for such a quick response :) Bare with me, although I did get this to work, I will not profess to be excellent at this stuff..Very beginner.  As far as the LAN layout, I have a router (actually a few daisy-chained) which works great on all the devices plugged in.  The source of the internet is from a cable modem.  So, the MAC is plugged into via ethernet, as well as the Raspberry Pi.  My MAC is plugged into a whole house ethernet outlet on the wall (which comes from the router.)  The Raspberry Pi I had plugged into via ethernet both ways (via an ethernet outlet on a wall, as well as plugged directly into one of the routers.)  I don't think I have any additional access point device?  I do have Wifi hooked up to the router, but I am not using that method to supply the internet pipe (I am only using wifi on my iOS.)  

Thanks ;)

**Comment by cflurin** [comment]
I would try using the Wifi of your Mac. This way the IOS device is connected via Wifi.
I would also add a Wifi Adapter (Dongle) to your RPi.
If your are using homebridge on the Mac and RPi, you should define two different name and username for the bridge in the config.json.

**Comment by raspman** [comment]
I see.  So hard wired Ethernet doesn't work?  That's strange, because I got the hard wired Ethernet to work on my Mac.  My iPhone was connected via wifi and it worked ok.  

Are you saying that connecting the raspberry to the wifi makes a difference?  That seems odd, as I would think as long as it's connected to the network, that should be fine?  

Also, I'm not running the Mac and raspberry simultaneously for home bridge. My goal is to use only the raspberry for this, but, of course, I can't get it to show up for Eve to work.

**Comment by cflurin** [comment]
Take a look [here](https://github.com/KhaosT/HAP-NodeJS/issues/108).
You can also find more infos on https://github.com/nfarina/homebridge.

I'm running homebridge on a Mac mini (Wifi) and a RPi 2 jessie with a Wifi dongle.

**Comment by raspman** [comment]
Thanks. I'll definitely try the single. And my Mac might be connected to both Ethernet and wifi :). I have to double check.

**Comment by raspman** [comment]
Single = wifi  dongle. Sorry typo

**Comment by raspman** [comment]
Thank you!  I got the wifi dongle and it immediately solved the problem!!  Just a question for you, if you don't mind (I think I saw it, but since you are the expert...)  How do I automatically run home bridge in the background on the Raspberry?  As it stands now, I ran the home bridge application via ssh from my OSx.  Now, if I got to close the terminal, the home bridge shuts down...id like to have it running in the background with the terminal running...thank you :)
